src/components/Dashboard/TalkDashboard.vue

```diff
@@ -524,7 +524,7 @@ function scrollEventCards({ direction }: { direction: 'backward' | 'forward' })
 	flex-grow: 1;
 
 	&> div {
```

Comment:
_Nit:_ should be `& > div` with whitespace, weird that stylelint doesn't pick it up